src/main/java/org/opentripplanner/model/plan/Itinerary.java

```diff
@@ -247,7 +247,7 @@ public String toStr() {
     for (Leg leg : legs) {
       buf.sep();
       if (leg.isWalkingLeg()) {
-        buf.walk((int) leg.getDuration());
+        buf.walk((int) leg.getDuration().toSeconds());
```

Comment:
It might make sense to add methods that take in a `Duration` in `PathStringBuilder`. But this can be done later if needed